### DISCUSSION TITLE
Change math font

### DISCUSSION THREAD

**Discussion by yordiak** [typst/typst#2230] [Q&A]
I would like to change the math font. However, I cannot find any way to do it on the documentation/tutorial.

Specifically, I want to use the 'Tex Gyre Pagella' font. Even though the font works for the text, it does not work for maths. I tried also the 'Tex Gyre Pagella Math' without any success.

**Comment by laurmaedje** [comment]
`#show math.equation: set text(font: "TeX Gyre Pagella Math")` does what you want. This should be added to the docs...

**Reply by abnerhexu** [reply]
Hey, that's cool. But when I changed this, I found the italic style of equations disappears. Is there any further supports for that? Thanks.
